src/main/java/org/team1540/robot2024/Robot.java

```diff
@@ -152,18 +148,15 @@ public void disabledPeriodic() {
     }
 
     public void enabledInit() {
```

Comment:
Ok so if these fancy exit methods exist why do we need enabled init when we can use the builtin disabledExit?

Reply:
ok I did not see this coming